shared/common-adapters/button.desktop.js

```diff
@@ -47,13 +131,79 @@ const buttonCommon = {
   minWidth: 10
 }
 
-export const styles = {
+const buttonCommon2 = {
+  ...globalStyles.DZ2.fontSemibold,
+  color: globalColorsDZ2.white,
+  borderRadius: 55,
+  fontSize: 16,
+  height: '32px',
```

Comment:
note: the things where you do have to put a unit: https://facebook.github.io/react/tips/style-props-value-px.html